Pipfile

```diff
@@ -24,17 +24,22 @@ boto3 = "*"
 pexpect = "*"
 pysftp = "*"
 setuptools = "*"
-cryptography = "==3.3.2"
+cryptography = "==3.4.8"
```

Comment:
The previous version would not compile on my local.  This is a minor change so it should be backwards compatible with what we were using.

Also since this is  --dev only requirement it wont affect production.

Reply:
@dkinzer This update caused us significant problems in the past.  See this issue for more details: https://tulibdev.atlassian.net/browse/DEVO-370

Can you confirm that this problem is resolved?

Reply:
And that is --dev here, but we do have that pinned in the airflow playbook.

Reply:
@sensei100 interesting.  I wonder why it's working now.  That issues claims that it was failing ant that we needed to look into making some changes to "our alpine" images.  So was this failing on cirlce?  I wonder why it's passing in my PR.